### ISSUE TITLE
Update AngularUI Bootstrap dependency

### ISSUE THREAD

**Issue by icem** [rpocklin/ui-router-tabs#26]
AngularUI was updated to 0.13.0 recently.

**Comment by icem** [comment]
Actually the real problem is that bower dependencies could not be resolved at all:
```
Unable to find a suitable version for angular, please choose one:
    1) angular#>=1 <1.3.0 which resolved to 1.2.28 and is required by angular-bootstrap#0.12.1
    2) angular#^1.3.15 which resolved to 1.3.15 and is required by angular-ui-router-tabs
    3) angular#1.3.15 which resolved to 1.3.15 and is required by angular-mocks#1.3.15
    4) angular#>= 1.0.8 which resolved to 1.3.15 and is required by angular-ui-router#0.2.14
```

**Comment by rpocklin** [comment]
Thanks, this went unnoticed when angular-ui-router was bumped to **0.2.14**.

That's great that AngularUI has been released, since it addresses bugs on the tabs component this directive was waiting on :)

For now, i've fixed the resolution you pointed out in `1.4.3`
